### DISCUSSION TITLE
Thanks for taking the time to put this handy little piece of software together

### DISCUSSION THREAD

**Discussion by JayRupp** [T-Troll/alienfx-tools#416] [General]
I can't express how much happier I am using AlienFX Tools. AWCC is a nightmare in every sense of the word. I feel like my computer has been freed from some bizarre form of captivity. Again, thank you for giving everyone a second (and better) option. Cheers, mate.

**Comment by T-Troll** [comment]
Thanks. In fact, i'm Alienware user for about 14 years, and i was sad to see latest AWCC lost a lot of useful features but consume more and more resources. So i remake it into 2010s style.
